src/pages/Dashboard/MyAccount/MyAccountUi.tsx

```diff
@@ -0,0 +1,217 @@
+            {t('myAccount.borrowBalance')}
+          </Typography>
+
+          {readableBorrowBalance || '-'}
+        </Typography>
+      </ul>
+
+      <div css={[styles.row, styles.topProgressBarLegend]}>
+        <div css={styles.inlineContainer}>
+          <Typography component="span" variant="small2" css={styles.inlineLabel}>
+            {t('myAccount.borrowLimitUsed')}
+          </Typography>
+
+          <Typography component="span" variant="small1" color="text.primary">
+            {readableBorrowLimitUsedPercentage || '-'}
+          </Typography>
+        </div>
+
+        <div css={styles.inlineContainer}>
+          <Typography component="span" variant="small2" css={styles.inlineLabel}>
+            {t('myAccount.limit')}
+          </Typography>
+
+          <Typography component="span" variant="small1" color="text.primary">
+            {typeof borrowLimitCents === 'number'
+              ? formatCentsToReadableValue(borrowLimitCents, true)
+              : '-'}
+          </Typography>
+        </div>
+      </div>
+
+      <ProgressBarHorizontal
+        css={styles.progressBar}
+        value={borrowLimitUsedPercentage || 0}
+        mark={safeBorrowLimitPercentage}
+        step={1}
+        ariaLabel="Borrow limit"
+        min={0}
+        max={100}
+        trackTooltip={
+          readableBorrowBalance &&
+          readableBorrowLimitUsedPercentage && (
+            <Trans
+              i18nKey="myAccount.progressBar.borrowLimitTooltip"
+              values={{
+                borrowBalance: readableBorrowBalance,
+                borrowLimitUsedPercentage: readableBorrowLimitUsedPercentage,
+              }}
+            >
+              Current borrow balance:
+              <br />
+              {readableBorrowBalance} ({readableBorrowLimitUsedPercentage} of your borrow limit)
+            </Trans>
+          )
+        }
+        markTooltip={
+          readableSafeBorrowLimit &&
+          safeBorrowLimitCents &&
+          safeBorrowLimitCents > 0 && (
+            <Trans
+              i18nKey="myAccount.progressBar.safeBorrowLimitTooltip"
+              values={{
+                safeBorrowLimit: readableSafeBorrowLimit,
+                safeBorrowLimitPercentage,
+              }}
+            >
+              Safe borrow limit:
+              <br />
+              {readableSafeBorrowLimit} ({safeBorrowLimitPercentage}% of your borrow limit)
+            </Trans>
+          )
+        }
+        isDisabled
+      />
+
+      <Typography component="div" variant="small2" css={styles.bottom}>
+        <Icon name="shield" css={styles.shieldIcon} />
+
+        <Typography component="span" variant="small2" css={styles.inlineLabel}>
+          {t('myAccount.safeLimit')}
+        </Typography>
+
+        <Typography component="span" variant="small1" color="text.primary" css={styles.safeLimit}>
+          {readableSafeBorrowLimit || '-'}
```

Comment:
This is the same duplicated logic where we check if a value is a number (in this case truthy), if it is we do something, if it isn't we return a `-`